packages/astro/src/runtime/server/render/component.ts

```diff
@@ -391,7 +392,8 @@ If you're still stuck, please open an issue on GitHub or join us at https://astr
 					})
 				);
 			}
-			destination.write(markHTMLString(renderElement('astro-island', island, false)));
+			const renderedElement = renderElement('astro-island', island, false);
+			destination.write(renderedElement);
```

Comment:
I think we still need to call `markHTMLString` since `renderElement` doesn't

Reply:
That's right! I left it out accidentally.